lib/src/drawings.dart

```diff
@@ -312,15 +336,21 @@ class BlankScreen extends GYWDrawing {
 
   const BlankScreen({this.color});
 
+  @override
+  GYWDrawing invertLightness() {
+    // FIXME(roscale): It doesn't make sense to have nullable background color.
```

Comment:
Yes, it does. It means that the screen background color must not be modified, but that the screen content must still be cleared.
To give you an actual use case, you can have items that are of a particular type and to highlight that, you use a particular background color. We can perform several applications on this item, and those actions may be split in several screens. To avoid writing the check in multiple screen, we simply do not add a background color so that this particular color stays as background until we reach a new item that will change the color.